Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormFormationDate.cshtml

```diff
@@ -62,7 +62,7 @@
                             <div class="govuk-date-input__item">
                                 <div class="govuk-form-group">
                                     <label class="govuk-label govuk-date-input__label" for="registrationDateDay">
-                                        Day
+                                        @StaticTextResource.Supplier_LegalFormFormationDate_Day
```

Comment:
We have `Global_Day` etc which can be used for these

Reply:
Done